src/test/utilities/mockito.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as mockito from 'ts-mockito'
+import { Mocker } from 'ts-mockito/lib/Mock'
+
+// Workaround https://github.com/NagRock/ts-mockito/issues/163
+export function mock<T>(): T {
+    const mocker = new Mocker(undefined)
+    mocker['excludedPropertyNames'] += 'then'
+
+    return mocker.getMock()
+}
+
+export const spy = mockito.spy
```

Comment:
why abstract this?

Reply:
Thanks, changed to `export {spy} from 'ts-mockito'`